Chap_Terms.tex

```diff
@@ -20,23 +20,30 @@ \chapter{PMIx Terms and Conventions}
+\item \declareterm{job} refers to a set of one or more \emph{applications} executed as a single invocation by the user within a session with a unique identifier, the \emph{job ID}, assigned by the \ac{RM} or launcher. For example, the command line ``\textit{mpiexec -n 1 app1 : -n 2 app2}'' generates a single \ac{MPMD} job containing two applications. A user may execute multiple \emph{jobs} within a given session, either sequentially or concerrently.
 
-\item \declareterm{application} refers to a single executable (binary, script, etc.) member of a \emph{job}.
+\item \declareterm{namespace} refers to a character string value assigned by the \ac{RM} to a \textit{job}.  The requester of the \textit{job} (e.g., \code{mpiexec} or a \ac{PMIx} client) may request a specific \emph{namespace} value.  All \textit{applications} executed as part of that \textit{job} share the same \emph{namespace}. The \emph{namespace} assigned to each \emph{job} must be unique within the scope of the governing \ac{RM} and often is implemented as a string representation of the numerical emph{Job ID}. The \emph{namespace} and \emph{job} terms will be used interchangeably throughout the document.
+
+\item \declareterm{application} represents a set of identical, but not necessarily unique,
+execution contexts within a \emph{job}.
 
 \item \declareterm{process} refers to an operating system process, also commonly referred to as a \emph{heavyweight} process. A process is often comprised of multiple \emph{lightweight threads}, commonly known as simply \declaretermAlt{threads}{thread}.
 
-\item \declaretermAlt{slot}{slots} refers to an allocated entry for a process. \acp{WLM} frequently allocate entire nodes to a \emph{session}, but can also be configured to define the maximum number of processes that can simultaneously be executed on each node. This often corresponds to the number of hardware \acp{PU} (typically cores, but can also be defined as hardware threads) on the node. However, the correlation between hardware \acp{PU} and slot allocations strictly depends upon system configuration.
+\item \declaretermAlt{slot}{slots} refers to an allocated set of resources assigned to a process.  \acp{WLM} frequently allocate entire nodes to a \emph{session}, but can also be configured to define the maximum number of processes that can simultaneously be executed on each node. This often corresponds to the number of hardware \acp{PU} (typically cores, but can also be defined as hardware threads) on the node. However, the correlation between hardware \acp{PU} and slot allocations strictly depends upon system configuration.
```

Comment:
"\acp{WLM} frequently allocate entire nodes to a \emph{session}"  I don't know what the macro \acp generates, but this might be grammatically incorrect. Should it be "WLMs frequently allocate" or "A WLM frequently allocates"?

Also, suggest changing "frequently" to  "typically" here, since 'typically' means what is usually done and I think 'frequently'  implies how often this happens (as in a rate)

Reply:
\acp{WLM} will expand to WLMs

I will make the change on frequently vs typically.   Good call.